webapp/client/src/components/FormFieldCheckBox.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import checkedIcon from "../assets/icons/checked.svg";
+
+const CheckBox = styled.div`
+  & {
+    padding: 0;
+    margin-top: var(--spacing-02);
+    flex-wrap: inherit;
+  }
```

Comment:
I think you can get rid of the `& {` part and just write 
```suggestion
padding: 0;
margin-top: var(--spacing-02);
flex-wrap: inherit;
```

Reply:
Yes, you are right. I removed some classes and forgot to remove &